src/concepts/pipeline/data_pipeline_detail.md

```diff
@@ -237,7 +237,7 @@ There is a command-line interface to STMO called [St. Mocli], if you prefer writ
 
 ##### Databricks: Managed Spark Analysis
 
-Our [Databricks instance] (see [Databricks docs]) offers another notebook interface for doing analysis in Scala, SQL, Python and R.
+Our Databricks instance (see [Databricks docs]) offers another notebook interface for doing analysis in Scala, SQL, Python and R.
```

Comment:
Can you just delete this, while you're here?

Reply:
That page is part of "Historical Reference" here: https://docs.telemetry.mozilla.org/concepts/pipeline/data_pipeline_detail.html#databricks-managed-spark-analysis

Reply:
Ah got it, I guess this predates us being able to do redirects. Just unlinking things sounds fine then. 👍